### DISCUSSION TITLE
Enhance FreeSewing's descriptions on Social Media

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#1415] [Forum]
I mentioned a while ago that FreeSewing's description on social media ("FreeSewing is an open source platform for made-to-measure sewing patterns.") was a bit technical and doesn't really invite newcomers. Relevant: [jargon turns people away](https://www.sciencedaily.com/releases/2020/02/200212084357.htm).

@joostdecock mentioned he wanted to keep the term "open-source" in there, because that explains why the platform is free; it's built on passion.

Suggestions:
- My initial suggestion was to start off with "free sewing patterns" (or something along those lines), that's clear and easy to understand, and then you can add more technical terms. Absolutely not suggesting that "free sewing patterns" should be the only/main thing of the description and that the rest would be up to the user to explore.
- Describing it as "the Wikipedia of sewing patterns", since that's a commonly known term.
- @/shieldmaiden mentioned this in today's call: first the catchphrase of "sewing patterns for non-average people; average people don't exist" and then a brief description.
- Something along the lines of what I did with [the Disboard description](https://disboard.org/server/698854858052075530).

**Note:** these suggestions aren't fleshed out much, because I haven't taken the time to properly brainstorm, but I felt I should still open the issue since it was mentioned last meeting #1191 
I will comment with more exact suggestions after brainstorming.

**Comment by joostdecock** [comment]
On the website, we've been using this for a while now:

```
Come for the sewing patterns
Stay for the community
```

**Comment by l3xander** [comment]
Perhaps we should turn this into a discussion? @joostdecock 

Anyways, my thoughts:
> Sewing patterns made by the community, for the community! Drafted to your measurements through code, all within your browser. At its core, FreeSewing is an open-source software project and our goal is to become the Wikipedia of sewing patterns! Made possible by our volunteers' time and effort, and funded by patrons.

(Is that the correct way to express the platform, "At its core, FreeSewing is.."? Otherwise I can replace "At its core," with "In technical terms,") 

Instagram and Twitter are fond of emojis. So, another option would be:
> 👕 Sewing patterns by the community, for the community!
> 🧍‍♂️ Drafted to your measurements through code, all within your browser
> 🤓 Open-source software project
> 🌐 Goal: to become the Wikipedia of sewing patterns
> ♥ Funded by patrons and our volunteers' time and effort

The descriptions could be considered too long. 🤔 I am not the right person when it comes to short sentences.
(Must mention that I am on Windows and do not have access to the gender-neutral emojis, those would be preferred.)

I've seen some people say that a description should end with a 'call for action', like "come join us today!" but that does not feel natural to me 😂 We're also not commercial. But, thoughts? Could be as simple as "Join our Discord chatrooms with the link below"

Feel free to be nitpicky.

**Comment by joostdecock** [comment]
Ok, converted to discussion.

I'm ok with emojis, I think it's a good way to keep it short. People don't read anymore :wink: 
If anything, I think it's a bit long still. If I am allowed to nitpick on each line:

> :shirt:  Sewing patterns by the community, for the community!

I feel this *by the community, for the community* thing is beaten to death. So I would only keep *sewing patterns* from this.

> :standing_man: Drafted to your measurements through code, all within your browser

The people for whom *within your browser* is meaningful are probably perfectly capable to figure that out on their own, so I don't think that adds much. And *through code* sounds scare for the majority of people who have no idea how any of this works.
As for *drafted to your measurements*, I'm not a fan of that because not everybody know what drafting is, which is why I typically just use *made-to-measure* since it's more common. 

If I combine the first two lines, that gives me:

- :shirt: Made-to-measure sewing patterns

> :nerd_face: Open-source software project

Just a nitpick on the spelling: it's `open source` not `open-source`

> :globe_with_meridians: Goal: to become the Wikipedia of sewing patterns

I know that I'm the one who coined that phrase, and I think it makes sense. But devout of context like this I worry it communicates that we're just trying to catalog the sewing patterns that are out there in the world (think PatternReview). As such, I feel that what it potentially adds it takes away by being ambiguous so I'd just drop it.

> :hearts: Funded by patrons and our volunteers' time and effort

Funded is factually incorrect. I'd prefer something like:

- :construction_worker_woman: Built by our contributors & supported by our patrons :heart: 


If I summarize that and throw some end emojis in for consistency, I get:

- :shirt: Made-to-measure sewing patterns :tada: 
- :nerd_face: Open source software project :rocket: 
- :construction_worker_woman: Built by contributors & supported by patrons :heart: 

Rules of 3: Always good :)

**Reply by l3xander** [reply]
I suggest we do this as our description instead of having the Discussion sit around for forever.
I'll change it on Instagram. @joostdecock, can you change it on Twitter?

**Reply by l3xander** [reply]
I can't figure out how to mark this as Answer 🤔

**Reply by l3xander** [reply]
Locking it is an option.

**Comment by l3xander** [comment]
Thank you @joostdecock! 😄

So, that is more or less the initial description but in emoji format.

What I have in mind is the people who aren't knowledged in these terms. The average person won't know of "open source" or "made-to-measure".
Or if I estimated wrong and the average person does know of these terms, there will still be a lot of people that don't understand, and people don't want to do difficult things in their free time. I do not mean that we should completely remove technical terms ((e.g. we keep "open source")) or that we should spell it out to them, that is too much, but a little bit of explaining/adding "easy words" would help with reaching those people too. Making things as clear as possible.

- I would argue that "made-to-measure" is not a commonly known word, at least not to the average person. Instead of "drafted", could do "generated", or just "made". (The difference between "made to your measurements" and "made-to-measure" is small but for the average person I think it's big.(?))
- Right now it gives the impression that you have to download something ("software project"). I added the "within your browser" to suggest that it is easily accessible; all you need is a device with WiFi connection.
- I added "through code" to make clear that the patterns are not manually drafted by people, but that it's basically automatic.
- What about "building a library from/for our own patterns"? It may clutter, but I like the impression that it gives.

And there are a lot of ways to do this, the above suggestions are not be-all and end-all. 
I'd say we find what we agree on to be short and clear?
What do you think?

Other than that 👍👍👍

**Reply by l3xander** [reply]
Should note that the description is not _that_ vague. Just, jargon.

**Comment by l3xander** [comment]
Could also be that I'm too fixated on one thing 😉
In the perfect and ideal description (which I must note doesn't exist), all of these points would be made clear:
- Built by contributors. Community. Patrons.
- Based on code. Platform behind it too. All open source.
- Drafted to personal measurements (or any measurements you put in). Customizable.
- Easy to access, online.
- Goals of becoming Wikipedia/building a library. Includes sewing guides and the like.
- (Privacy friendly. Fuck capitalism.)

And we (I) can make a 'permanent Instagram story' to further explain these points, but I feel like the description should be clear right off the bat.

**Comment by tangerineshark** [comment]
I don't have a ton of opinions on the language, although I think "made-to-measure" is clear enough. But as a fun accessibility heads-up, if you put your emoji at the end of the line (or tweet or sentence or whatever), it's nice for folks using screenreaders. ❤️

**Comment by joostdecock** [comment]
For people interested in accessibility, this us how you write an emoji:

```html
<span role="img" aria-label="Cup of coffee">☕</span>
```

Obviously, that requires you can input HTML.

Apart from that, do we just summarize it all as **Fuck capitalism**? 😂

**Reply by tangerineshark** [reply]
So simple, so elegant

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
